ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Util.hs

```diff
@@ -103,38 +102,12 @@ readAll HasFS{..} hnd = go mempty
   where
     bufferSize = 64 * 1024
     go acc = do
-      bytesRead <- hGet hnd bufferSize
+      bytesRead <- hGetSome hnd bufferSize
       let acc' = acc <> BS.byteString bytesRead
       case BS.length bytesRead < bufferSize of
         True  -> return acc'
         False -> go acc'
 
--- | Variant of 'hGet' that throws an 'FsReachedEOF' 'FsError' when the gotten
--- number of bytes didn't match the number of request bytes.
---
--- TODO move to Ouroboros.Storage.FS.Class?
```

Comment:
Good that we could get rid of this.

Can you check that there are no unused imports after removing it, e.g. `ByteString`?